cpp/example_code/rds/getting_started_with_db_instances.cpp

```diff
@@ -0,0 +1,1079 @@
+                      << std::endl;
+        }
+        else {
+            std::cerr << "Error with RDS::CreateDBParameterGroup. "
+                      << outcome.GetError().GetMessage()
+                      << std::endl;
+            return false;
+        }
+        // snippet-end:[cpp.example_code.rds.CreateDBParameterGroup]
+    }
+
+    printAsterisksLine();
+    std::cout << "Let's set some parameter values in your parameter group."
+              << std::endl;
+
+    Aws::String marker;
+    Aws::Vector<Aws::RDS::Model::Parameter> autoIncrementParameters;
+    // 4.  Get the parameters in the DB parameter group.
+    if (!getDBParameters(PARAMETER_GROUP_NAME, AUTO_INCREMENT_PREFIX, NO_SOURCE,
+                         autoIncrementParameters,
+                         client)) {
+        cleanUpResources(PARAMETER_GROUP_NAME, "", client);
+        return false;
+    }
+
+    Aws::Vector<Aws::RDS::Model::Parameter> updateParameters;
+
+    for (Aws::RDS::Model::Parameter &autoIncParameter: autoIncrementParameters) {
+        if (autoIncParameter.GetIsModifiable() &&
+            (autoIncParameter.GetDataType() == "integer")) {
+            std::cout << "The " << autoIncParameter.GetParameterName()
+                      << " is described as: " <<
+                      autoIncParameter.GetDescription() << "." << std::endl;
+            if (autoIncParameter.ParameterValueHasBeenSet()) {
+                std::cout << "The current value is "
+                          << autoIncParameter.GetParameterValue()
+                          << "." << std::endl;
+            }
+            std::vector<int> splitValues = splitToInts(
+                    autoIncParameter.GetAllowedValues(), '-');
+            if (splitValues.size() == 2) {
+                int newValue = askQuestionForIntRange(
+                        Aws::String("Enter a new value in the range ") +
+                        autoIncParameter.GetAllowedValues() + ": ",
+                        splitValues[0], splitValues[1]);
+                autoIncParameter.SetParameterValue(std::to_string(newValue));
+                updateParameters.push_back(autoIncParameter);
+
+            }
+            else {
+                std::cerr << "Error parsing " << autoIncParameter.GetAllowedValues()
+                          << std::endl;
+            }
+        }
+    }
+
+    {
+        // 5.  Modify the auto increment parameters in the group.
+        // snippet-start:[cpp.example_code.rds.ModifyDBParameterGroup]
+        Aws::RDS::Model::ModifyDBParameterGroupRequest request;
+        request.SetDBParameterGroupName(PARAMETER_GROUP_NAME);
+        request.SetParameters(updateParameters);
+
+        Aws::RDS::Model::ModifyDBParameterGroupOutcome outcome =
+                client.ModifyDBParameterGroup(request);
+
+        if (outcome.IsSuccess()) {
+            std::cout << "The DB cluster parameter group was successfully modified."
```

Comment:
This comment says cluster, but I'm not sure this is a cluster operation, since it isn't an Aurora example. Same for below where it mentions Cluster in the error message.